src/index.ts

```diff
@@ -146,3 +146,5 @@ export type {
   SpeedDialProps,
   ThemeProps,
 };
+export { default as Tab } from './tab/tab';
```

Comment:
Why not follow the pattern of the rest of the file? Please use the same pattern as every other component.